test_calculus.py

```diff
@@ -66,3 +66,9 @@ def test_trapezoid_scipy():
     Unit test for scipy implementation of trapezoid method
     '''
     assert np.isclose(calc.trapezoid_scipy(np.sin, 0, np.pi), 2)
+
+def test_trapezoid_python():
```

Comment:
This unit testing could be more robust. It would be good to test if your function handles the exception cases well, such as a test for if your function handles the cases where the integrand is infinite at any point, wether upper limit, lower limit, or in-between.

Reply:
Incorporated a unit test for error handling too. Compared with the expected output, the test passes. Good suggestion, thanks. I had kept the unit test simple as recalled @laserlab discussing in a class to keep the unit tests minimal.